app/Shell/Shell.php

```diff
@@ -59,4 +65,48 @@ public function buildProcess(string $command): Process
 
         return $process;
     }
+
+    /**
+     * @param $buffer
+     * @return string
+     */
+    protected function formatMultiline($buffer): string
+    {
+        $bufferArray = explode(PHP_EOL, $buffer);
+        $buffer = $this->alignMultiline($bufferArray);
+        return implode(PHP_EOL, $buffer);
+    }
+
+    /**
+     * @param $buffer
+     * @return bool
+     */
+    protected function isMultiline($buffer): bool
+    {
+        return substr_count($buffer, "\n") > 1;
+    }
+
+    /**
+     * @param $spaces
+     * @return mixed
+     */
+    protected function indentBy($spaces)
+    {
+        return str_repeat(' ', $spaces);
+    }
+
+    /**
+     * @param bool $bufferArray
+     * @param array $buffer
+     * @return array
+     */
+    protected function alignMultiline(array $bufferArray): array
+    {
+        $buffer[0] = $bufferArray[0];
+        unset($bufferArray[0]);
+        foreach ($bufferArray as $line) {
+            $buffer[] = $this->indentBy(self::SPACES) . $line;
+        }
+        return $buffer;
```

Comment:
I think it would be cleaner if we used collections here:

```php
return collect($bufferArray)->map(function ($item, $index) {
    return $index > 0 ? Str::of($item)->prepend(str_repeat(' ', self::SPACES)) : $item;
})->toArray();
```